torch/_torch_docs.py

```diff
@@ -181,15 +155,15 @@ def merge_dicts(*dicts):
 .. math::
     \text{{out}} = \text{{input}} + \text{{alpha}} \times \text{{other}}
 
-If :attr:`other` is of type FloatTensor or DoubleTensor, :attr:`alpha` must be
-a real number, otherwise it should be an integer.
+If the result of adding :attr:`self` and :attr:`other` would not have a
```

Comment:
this isn't right after @nairbv's changes (also with sub below).  More accurate is something like "the category (floating point, integer, boolean)" of alpha must match that of the result of addition between `input` and `other`.